### PR TITLE
Log full filtered world state as SSL_WrapperPacket

### LINKED ISSUE #1898: Define a Protobuf message for the full filtered world state and log it
### Description of the task

Define a Protobuf message for the full filtered world state and log it. The full world state should be reconstructible from this Protobuf message.

### Acceptance criteria

- [ ] Define Protobuf message for filtered world state
- [ ] Log it to a file
- [ ]

### PR DESCRIPTION
**Pull request by jordanzliu**
### Description

Log full filtered world state as SSL_WrapperPacket

### Testing Done

Unit tests

### Resolved Issues

Resolves #1898 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/software/proto/message_translation/ssl_wrapper.cpp [hunk 1]
```diff
@@ -16,3 +21,40 @@ std::unique_ptr<SSLProto::SSL_WrapperPacket> createSSLWrapperPacket(
 
     return std::move(wrapper_packet);
 }
+std::unique_ptr<SSLProto::SSL_WrapperPacket> createSSLWrapperPacket(
+    const World& world, bool friendly_team_colour_yellow)
```
Comment (jonathanlew) [inline src/software/proto/message_translation/ssl_wrapper.cpp]: we have a TeamColour enum that you can use

**Review by jonathanlew — CHANGES_REQUESTED** [review]
looks good just a nit

**Review by jonathanlew — APPROVED** [review]
LGTM!

**Review by kelvinkoon — APPROVED** [review]
💯

**Pull request merged**